front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainsListV2.tsx

```diff
@@ -49,6 +49,18 @@ const ImportTrainScheduleTrainsListV2 = ({
     [rollingStocks]
   );
 
+  // Format the trains list to match the train names provided by the backend
+  const formattedTrains = useMemo(
```

Comment:
what about formattedTrainList ?

Reply:
Renamed it to formattedTrainsList

Reply:
nice, just about train**s**list, it seems to me better to have trains OR trainList (list refer to an array), but it's only a proposition

Reply:
In most of the app, we have an "s" even with a list const, so I think we should keep it with `trainsList`. But I agree that a list can be empty or have just one item inside so the "s" might not be relevant. It can be discuss in a front meeting (even though it will be painfull to rename all the related consts in the app)